### PR TITLE
sync auth process

### PR DESCRIPTION
**Pull request by Sereg20**

### CODE REVIEW

File: uimodule/webapp/Firebase.js [hunk 1]
```diff
@@ -26,13 +33,18 @@ sap.ui.define([
             usersRef.on("value", snapshot => {
```
Comment (bahachou) [inline uimodule/webapp/Firebase.js:33]: hung notation

File: uimodule/webapp/Firebase.js [hunk 2]
```diff
@@ -26,13 +33,18 @@ sap.ui.define([
...
                 this.handleOnMethodSuccess(snapshot.val());
             }, error => {
-                console.log("Error: " + error.code);
+                fbLoadPromise = Promise.resolve(error);
             });
 		},
 
         handleOnMethodSuccess(snapshotData) {
```
Comment (bahachou) [inline uimodule/webapp/Firebase.js:40]: hung notation

File: uimodule/webapp/Firebase.js [hunk 3]
```diff
@@ -26,13 +33,18 @@ sap.ui.define([
...
             const fbModel = sap.ui.getCore().byId("container-ohranaTryda---idAppControl").getModel();
+            const activeUserMobile = sessionStorage.getItem("ActiveUserMobile");
+            const activeUser = snapshotData.find(user => user.mobilePhone == activeUserMobile);
+
             fbModel.setProperty("/Users", snapshotData);
+            activeUser ? fbModel.setProperty("/ActiveUser", activeUser) : null;
```
Comment (bahachou) [inline uimodule/webapp/Firebase.js:46]: same

File: uimodule/webapp/controller/ListReport.controller.js [hunk 4]
```diff
@@ -10,7 +10,18 @@ sap.ui.define([
     return Controller.extend("bntu.ohranaTryda.controller.ListReport", {
         
         onInit() {
-            this.openLoginDialog();
+            
+        },
+
+        onAfterRendering() {
+            this.getFbLoadPromise().then(() => {
+                const sActiveUserMobile = this.getModel().getProperty("/ActiveUser").mobilePhone;
```
Comment (bahachou) [inline uimodule/webapp/controller/ListReport.controller.js:18]: n?

File: uimodule/webapp/Firebase.js (hunk 1 above)
Comment (Sereg20) [reply uimodule/webapp/Firebase.js:33]: It should not be in Firebase.js file I guess

**Pull request merged**